src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesReducer = createSlice({
```

Comment:
- [OPTIONAL] I highly recomment you to change the name of your Slice to something like `CategoriesSlice` to give it more sense, as it is not the reducer

Reply:
Hello @GedeonTS 

Thank you for your kind feedback. I shall make the changes and ask for a review again.

Regards